src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -173,21 +173,27 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ dispatch, editData }) => {
     </S.InputContainer>
   );
 
-  const renderBannerUpload = () => (
-    <S.InputContainer>
-      <h4 className="italic" style={{ fontSize: "20px" }}>
-        Upload a banner photo
-      </h4>
-      <ImageInput
-        onChange={handleBannerUpload}
-        name="bannerImage"
-        error={errors.bannerImageFile || errors.bannerImageUrl}
-        setValue={setValue}
-        register={register}
-        imgUrl={editData?.bannerImageUrl}
-      />
-    </S.InputContainer>
-  );
+  const renderBannerUpload = () => {
```

Comment:
looks like we renaming banner photo -> to hightlight image, can we rename that in the code as well ?

Reply:
renamed in b396ea2
should we also rename all the `banner` related props & db references to `highlight` ones? we'd need to have backward compatibility if so